app/javascript/components/helpers/Load.jsx

```diff
@@ -0,0 +1,30 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCircleNotch } from "@fortawesome/free-solid-svg-icons";
+
+/**
+ * @prop itemType: Displays loading animation overlay
+ */
+class Load extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    if (this.props.itemType) {
+      return (
+        <div className="w-100 bg-snow pa2 pt3 pb3 overlay-full-centered">
```

Comment:
Tachyons!! 😻 Let's try to reduce the amount of CSS we have in `overlay-full-centered`. There are a couple more Tachyons classes we can use here. It actually eliminates any need for another SCSS class 👀 

```suggestion
        <div className="w-100 bg-snow pa2 pt3 pb3 fixed z-1 bg-white-90 top-0 bottom-0 flex flex-column justify-center items-center">
```
It looks verbose, but I think the visual defect is worth having less SCSS floating about!

*Also don't think `right: 0`/`left: 0` is necessary but correct me if I am wrong~